services/order.py

```diff
@@ -0,0 +1,23 @@
+from django.db import transaction
+from db.models import Order, Ticket, MovieSession, User
+
+def create_order(tickets, username, date=None):
+    user = User.objects.filter(username=username).first()
+    if not user:
+        return None
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user, created_at=date) if date else Order.objects.create(user=user)

```

Comment:
The conditional logic for setting the `created_at` date in the `Order` creation can be simplified by using the `created_at` field's default behavior. Consider removing the `created_at=date` part and rely on the `auto_now_add=True` attribute in the model definition.